include/PianoRoll.h

```diff
@@ -385,6 +390,12 @@ protected slots:
 	int m_ppb;  // pixels per bar
 	int m_totalKeysToScroll;
 
+	static int s_keyLineHeight;
```

Comment:
Is there a reason this needs to be `static`?

Reply:
That's because `PianoRoll::drawNoteRect()` is `static`.

Reply:
I'm looking through and I don't see a need for `drawNoteRect` to be static either.
@PhysSong @tresf any insight on this?

Reply:
I think it should be a member variable because it's a property of the piano-roll instance. Now `PianoRoll::drawNoteRect()` should know the value of `s_keyLineHeight` because it's the height of the note to draw. We can either make `PianoRoll::drawNoteRect()` nonstatic or add a height parameter.

Reply:
I would prefer a solution with removing `static` from the `PianoRoll::drawNoteRect()` function (if using `static` in it is not intentional). This function already has a lot of params. 

I didn't know why `PianoRoll::drawNoteRect() is `static` so I did `keyLineHeight` static too.

Reply:
So I will remove the static modifier from `PianoRoll::drawNoteRect()` if no one protests.

Reply:
Fixed by https://github.com/LMMS/lmms/pull/5442/commits/c69f02677a280ab50b12914ff9747e1741dca4a6